### ISSUE TITLE
admin-ajax errors on multisite

### ISSUE THREAD

**Issue by jhigginsdoe** [opencredit/badgeos#439]
We've set up a multisite with BadgeOS.  When we click "Add New Step", nothing happens.  I see there is a javascript error message regarding admin-ajax.php.  This same process works fine on the main site, just not on the secondary site that we set up in wordpress.  The secondary site is in a subfolder on the same domain.  Any thoughts?  

Thanks,
Jason

**Comment by johardit** [comment]
Also having this problem on our main-site, but without an error message (see attached screenshots). There is a long-delay after pressing the add-step button and no label field appears for any step option. 

Any ideas? Could there be a plugin conflict? (I already deactivated/reactivated the add-ons).

Was really looking forward to using BadgeOS with Learndash, but I can't seem to get started because of this issue.

Thanks,

Johar
![2015-04-26_2314](https://cloud.githubusercontent.com/assets/12130327/7341176/c2b64e32-ec6a-11e4-93b6-aa57d641df9f.png)

**Comment by tw2113** [comment]
Do you have the Community add-on enabled but not BuddyPress? That's usually been the source of error for this one, if I recall right.

**Comment by johardit** [comment]
It would seem so! I deactivated the Community Add-on and it seems to work. If I wanted to succesfully enable the community add-on, am I to presume it would only function if I also have the BuddyPress plugin active too?

Thanks for the swift response!

**Comment by tw2113** [comment]
One of the open issues we have open for Community Add-on is that it shouldn't be activate-able without BuddyPress active since it is meant for it.

**Comment by johardit** [comment]
That's a good idea! I am using bbPress and the description suggested the community-add on was for it and/or BuddyPress, so thanks for clarifying!

**Comment by tw2113** [comment]
Closing this issue. It's Community Addon-specific and it's been amended in the community addon since.
